tests/cisco_ios/show_tacacs/cisco_ios_show_tacacs_status_fail_count.raw

```diff
@@ -0,0 +1,32 @@
+Tacacs+ Server -  public  :
+               Server name: TACACS_1
+            Server address: 10.20.30.1
+               Server port: 49
+              Socket opens:      21752
+             Socket closes:      21752
+             Socket aborts:          0
+             Socket errors:          0
+           Socket Timeouts:          0
+   Failed Connect Attempts:          52
```

Comment:
```suggestion
   Failed Connect Attempts:         52
```

Gah, cosmetic as the numbers are right aligned.
Yeah, I'm with you - the changes are getting old, heh.